subprojects/gradle/cd/src/main/kotlin/com/avito/ci/CiStepsPlugin.kt

```diff
@@ -0,0 +1,67 @@
+package com.avito.ci
+
+import com.avito.utils.gradle.BuildEnvironment
+import com.avito.utils.gradle.buildEnvironment
+import org.gradle.api.NamedDomainObjectContainer
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.Task
+import org.gradle.kotlin.dsl.container
+import org.gradle.kotlin.dsl.register
+
+class CiStepsPlugin : Plugin<Project> {
+
+    private val taskGroup = "ci"
+
+    override fun apply(project: Project) {
+
+        val buildContainer = project.container { name ->
+            BuildStepListExtension(name, project.objects)
+        }
+
+        project.extensions.add("builds", buildContainer)
+
+        buildContainer.registerPredefinedBuildTasks()
+
+        // TODO: MBS-6118. В CD во время конфигурации активно используется git. Это слишком долго.
```

Comment:
This task is closed a long time ago. Does this TODO still actual?

Reply:
Seems like it is outdated, but i won't enable this plugin for local runs yet